crabcache/templates/configMap2.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: crabcache
+  labels:
+    app: crabcache
+  namespace: crab
+data:
+  setup-certs-and-run.sh: |
+    #!/bin/bash
+    cp /host/etc/grid-security/* /etc/grid-security
+    echo 'INFO Files in /etc/grid-security'
+    ls -lahZ /etc/grid-security
+    su  -c "cd /data && /data/run.sh" --preserve-environment  _crabcache
```

Comment:
We don't need extra file for configmap. Can be included in a single file after `---` lines.